### ISSUE TITLE
Update rule description to the latest template

### ISSUE THREAD

**Issue by jelly** [oxc-project/oxc#5458]
In my [PR](https://github.com/oxc-project/oxc/pull/5389#discussion_r1743707376)  I was using the old template which I copied from another rule, the [new template](https://github.com/oxc-project/oxc/blob/1d3e973f101fc2d03f705f75c7de9e9cb04ae50a/tasks/rulegen/template.txt#L24) shows the Bad and Good examples different.

There are still rules around which use the old template i.e.:

```
    /// ### Example
    /// ```jsx
    /// // Bad
    /// <html />
    ///
    /// // Good
    /// <html lang="en" />
    /// ```
```

They can be found with:

```
git grep "\/\/ Bad"
```

And one uses a different template:

crates/oxc_linter/src/rules/typescript/no_useless_empty_export.rs:    /// ### Bad

**Comment by jelly** [comment]
This looks like a good first issue I'd say.

**Comment by shulaoda** [comment]
Yes, I am currently working on the docs of the relevant rules. You can refer to this PR #5428 for more details.
